core/ustring.cpp

```diff
@@ -3235,7 +3235,7 @@ static int _humanize_digits(int p_num) {
 String String::humanize_size(size_t p_size) {
 
 	uint64_t _div = 1;
-	static const char *prefix[] = { " Bytes", " KB", " MB", " GB", "TB", " PB", "HB", "" };
+	static const char *prefix[] = { " Bytes", " KB", " MB", " GB", " TB", " PB", " EB", "" };
```

Comment:
Just a note: if we wanted to continue this farther, there is ZB and YB, though these are beyond the current range of `uint64_t` for bytes.